common/src/provisioning/Authentication/ServiceConnectionString.cs

```diff
@@ -113,11 +126,20 @@ public string GetPassword()
             return password;
         }
 
+        /// <summary>
+        /// Returns the Authorization header value
+        /// </summary>
+        /// <returns></returns>
         public string GetAuthorizationHeader()
         {
             return GetPassword();
         }
 
+        /// <summary>
+        /// Parser for the Provisioning Service Connection String
+        /// </summary>
+        /// <param name="connectionString"> The DPS Connection String </param>
+        /// <returns></returns>
         public static ServiceConnectionString Parse(string connectionString)
```

Comment:
Code below here doesn't seem to make sense for a connection string parser. (e.g. Token creation).

Reply:
The token generation logic has been removed from the parser